Samples/Samples/View/CalendarPage.xaml.cs

```diff
@@ -13,14 +14,17 @@ public CalendarPage()
             InitializeComponent();
         }
 
-        void OnEventTapped(object sender, ItemTappedEventArgs e)
+        async void OnEventTapped(object sender, ItemTappedEventArgs e)
         {
             if (e.Item == null && e.Item is Event)
```

Comment:
use "e.Item is Event event" to define "event" that can be used below instead of (e.Item as Event)?.Id

Reply:
Altered in new commit